doc/engineering-reference/src/simulation-models-encyclopedic-reference-003/indoor-swimming-pool.tex

```diff
@@ -59,22 +59,22 @@ \subsection{Convection from the pool water surface}\label{convection-from-the-po
 The convection between the pool water and the zone are defined using:
 
 \begin{equation}
-Q_{conv} = h \cdot A \cdot (T_p – T_a)
+Q_{conv} = h \cdot A \cdot (T_p - T_a)
 \end{equation}
 
 \begin{equation}
-h = 0.22 \cdot (T_p – T_a)1/3
+h = 0.22 \cdot (T_p - T_a)^{\sfrac{1}{3}}
```

Comment:
@mjwitte @Myoldmopar Okay...on this one, I have no idea what the problem is.  I got this to work at one point and then it changed later even though I had not changed it.  I followed this based on some information on-line and it did work the first time I built the docs after making this change.  However, after that, it didn't matter what I did on my computer it kept saying 13.  I thought maybe it was a glitch on my computer but apparently not.  What is the correct way to do this?